src/main/flight/pid.c

```diff
@@ -167,7 +167,15 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .abs_control_limit = 90,
         .abs_control_error_limit = 20,
         .antiGravityMode = ANTI_GRAVITY_SMOOTH,
+        .dyn_lpf_dterm_max_hz = 200,
+        .dyn_lpf_dterm_idle = 20,
     );
+#ifdef USE_DYN_LPF
```

Comment:
Ok, this will work. I would suggest you reorder the `RESET_CONFIG()` call above so that the values changed here are at the bottom - this will make it easier to work with them in the future, as both variants are close to each other. Or else pull them out entirely, and put them into an `#else`.